code/modules/paperwork/paper.dm

```diff
@@ -222,28 +222,58 @@
 /obj/item/paper/attack_ai(var/mob/living/silicon/ai/user)
 	show_content(user)
 
-/obj/item/paper/attack(mob/living/carbon/M as mob, mob/living/carbon/user as mob)
-	if(user.zone_sel.selecting == BP_EYES)
-		user.visible_message("<span class='notice'>You show the paper to [M]. </span>", \
-			"<span class='notice'> [user] holds up a paper and shows it to [M]. </span>")
+/obj/item/paper/attack(mob/living/carbon/M, mob/living/carbon/user)
+	if (user.zone_sel.selecting == BP_EYES)
+		user.visible_message(
+			SPAN_NOTICE("You show the paper to [M]. "), \
+			SPAN_NOTICE("[user] holds up a paper and shows it to [M]. ")
+		)
 		M.examinate(src)
 
-	else if(user.zone_sel.selecting == BP_MOUTH) // lipstick wiping
-		if(ishuman(M))
+	else if (user.zone_sel.selecting == BP_MOUTH) // lipstick wiping
+		if (ishuman(M))
 			var/mob/living/carbon/human/H = M
-			if(H == user)
-				to_chat(user, "<span class='notice'>You wipe off the lipstick with [src].</span>")
+			if (H == user)
+				to_chat(user, SPAN_NOTICE("You wipe off the lipstick with [src]."))
 				H.lip_style = null
 				H.update_body()
 			else
-				user.visible_message("<span class='warning'>[user] begins to wipe [H]'s lipstick off with \the [src].</span>", \
-								 	 "<span class='notice'>You begin to wipe off [H]'s lipstick.</span>")
-				if(do_after(user, 2 SECONDS, H, do_flags = DO_DEFAULT & ~DO_BOTH_CAN_TURN))
-					user.visible_message("<span class='notice'>[user] wipes [H]'s lipstick off with \the [src].</span>", \
-										 "<span class='notice'>You wipe off [H]'s lipstick.</span>")
+				user.visible_message(
+					SPAN_WARNING("[user] begins to wipe [H]'s lipstick off with \the [src]."), \
+					SPAN_NOTICE("You begin to wipe off [H]'s lipstick.")
+				)
+				if (do_after(user, 2 SECONDS, H, do_flags = DO_DEFAULT & ~DO_BOTH_CAN_TURN))
+					user.visible_message(
+						SPAN_NOTICE("[user] wipes [H]'s lipstick off with \the [src]."), \
+						SPAN_NOTICE("You wipe off [H]'s lipstick.")
+					)
 					H.lip_style = null
 					H.update_body()
 
+	else if (user.zone_sel.selecting == BP_HEAD) // graffiti wiping
+		if (ishuman(M))
+			var/mob/living/carbon/human/H = M
+			var/obj/item/organ/external/head/head = H.organs_by_name[BP_HEAD]
+			if (!istype(head) || !head.forehead_graffiti)
```

Comment:
Put the least expensive checks first to save processing times (short circuit evaluation).

```suggestion
			if (!head.forehead_graffiti || !istype(head))
```

Reply:
Won't this result in a nullref if `head` isn't the right type?

Reply:
I am the big dumb. Ignore this.